packages/nodes-base/nodes/Google/Docs/GoogleDocs.node.ts

```diff
@@ -140,7 +143,7 @@ export class GoogleDocs implements INodeType {
 							//         replace values
 							// ----------------------------------
 
-							if (replaceAllTextValues.length) {
+							if (replaceAllTextValues && replaceAllTextValues.length) {
```

Comment:
Good catch. Ideally with the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining). Same for others.

Reply:
The `replaceAllTextValues.length` will not be reached if `replaceAllTextValues` is null or undefined, or do you mean we add it inside the IF body ?

Reply:
`if (replaceAllTextValues?.length) {`

In fact, this should also be reflected in the interface: `replaceAllTextValues?: Array<{`